gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-empty-state/api-documentation-v4-empty-state.component.ts

```diff
@@ -14,32 +14,28 @@
  * limitations under the License.
  */
 
-import { Component, EventEmitter, Input, OnDestroy, Output } from '@angular/core';
-import { Subject } from 'rxjs';
+import { booleanAttribute, Component, EventEmitter, Input, Output } from '@angular/core';
 
-import { getLogoForPageType, PageType, getTitleForPageType } from '../../../../../entities/management-api-v2';
+import { PageType } from '../../../../../entities/management-api-v2';
 
 @Component({
   selector: 'api-documentation-empty-state',
   templateUrl: './api-documentation-v4-empty-state.component.html',
   styleUrls: ['./api-documentation-v4-empty-state.component.scss'],
 })
-export class ApiDocumentationV4EmptyStateComponent implements OnDestroy {
-  private unsubscribe$: Subject<void> = new Subject<void>();
-
+export class ApiDocumentationV4EmptyStateComponent {
   @Output()
-  onAddPage = new EventEmitter<PageType>();
+  addPage = new EventEmitter<PageType>();
 
   @Input()
-  public isReadOnly = false;
+  isReadOnly = false;
+
+  @Input({ transform: booleanAttribute })
+  shouldShowAddPadeButton!: boolean;
```

Comment:
```suggestion
  @Input()
  showAddPageButton: boolean = false;
```

This uses a variable name a bit more "front-end-y" and sets an initial value to false.

Reply:
Done